installer/lib/phx_new/single.ex

```diff
@@ -42,7 +42,7 @@ defmodule Phx.New.Single do
   ]
 
   template :brunch, [
-    {:text, "phx_assets/brunch/gitignore",        :project, ".gitignore"},
+    {:eex,  "phx_assets/brunch/gitignore",        :project, ".gitignore"},
```

Comment:
I kept the current directory structure of `brunch` and `bare`, but I think we could merge both `gitignore` templates inside `phx_single/gitignore` and have an `if brunch do` to ignore `npm-debug.log` and `/node_modules/`.

Wdyt?

Reply:
Merging sounds good to me, since that's what we have done below anyway.